src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -0,0 +1,12 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<html>
+<head>
+    <style>
+        h1 {text-align: center;}
+    </style>
+    <title>my index page</title>
+</head>
+<body>
+<h1>Hello world!</h1>
```

Comment:
it will be great if we found the way to display the passed request attribute here)